Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 34, in resolve
    return self.call_controller(helper.get_package_from_module("controllers", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 57, in instance_controller
    return controller.action(self.request.action)
  File "/home/arthur/rep/uniwallet/framework/Controller.py", line 25, in action
    self.response.body = getattr(self, action_name)()
TypeError: arca() missing 1 required positional argument: 'html'

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 34, in resolve
    return self.call_controller(helper.get_package_from_module("controllers", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 57, in instance_controller
    return controller.action(self.request.action)
  File "/home/arthur/rep/uniwallet/framework/Controller.py", line 25, in action
    self.response.body = getattr(self, action_name)()
TypeError: arca() missing 1 required positional argument: 'html'

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 34, in resolve
    return self.call_controller(helper.get_package_from_module("controllers", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 57, in instance_controller
    return controller.action(self.request.action)
  File "/home/arthur/rep/uniwallet/framework/Controller.py", line 25, in action
    self.response.body = getattr(self, action_name)()
TypeError: arca() missing 1 required positional argument: 'html'

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

Traceback (most recent call last):
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 18, in route
    return self.resolve()
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 32, in resolve
    treater = self.call_controller(helper.get_package_from_module("controllers.treaters", "modules.{}".format(self.request.module)))
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 47, in call_controller
    return self.instance_controller(package, controller_name)
  File "/home/arthur/rep/uniwallet/framework/Router.py", line 55, in instance_controller
    mod = importlib.import_module(package.__name__ + ".{}".format(controller_name))
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/arthur/rep/uniwallet/modules/moises/controllers/treaters/noe.py", line 9
    "fields":{
           ^
SyntaxError: invalid syntax

